packages/swingset-liveslots/src/virtualObjectManager.js

```diff
@@ -1009,10 +1037,10 @@ export function makeVirtualObjectManager(
 
   function reanimateDurableKindID(vobjID) {
     const kindID = `${parseVatSlot(vobjID).subid}`;
-    const raw = syscall.vatstoreGet(`vom.dkind.${kindID}`);
-    raw || Fail`unknown kind ID ${kindID}`;
-    const durableKindDescriptor = JSON.parse(raw);
+    const durableKindDescriptor = loadDurableKindDescriptor(kindID);
+    const nextInstanceID = loadNextInstanceID(kindID);
```

Comment:
Belated feedback, but these functions are only used together and I wonder if they shouldn't be merged (which would also be convenient if the future includes batched reads).
```suggestion
    const { descriptor: durableKindDescriptor, nextInstanceID } = loadDurableKind(kindID);
```

On the other hand, maybe it is better to keep the {save,load}$Noun symmetry.